python-checks/src/test/resources/checks/tests/skippedTestNoReasonWithImport.py

```diff
@@ -0,0 +1,8 @@
+from pytest.mark import skip
+
+# Pytest
+class UsingPytest():
+    @skip()  # Noncompliant {{Provide a reason for skipping this test.}}
+#   ^^^^^^^
+    def test_pytest_mark_skip_no_reason():
```

Comment:
You could add a test for which the decorator is a name (`@skip`). Currently, it wouldn't be reported due to my other comment, but it probably should.

Reply:
I added this test case.
If for "my other comment" you refer to the HasSymbol part, then since I slightly adapted your proposition, this test case does report properly an error, which seems fine.